CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## [Unreleased]
 - Display granular error codes when unlock fails unexpectedly
+- Make bootloader mode more user-friendly
```

Comment:
Bootloader releases are separate from firmware releases (different versions and cycle) - maybe it's best to add a separate changelog or separate section for the bootloader in this changelog?